### PR TITLE
DOC Improve web worker documentation

### PR DESCRIPTION
**Pull request by cglacet**
Here is a first draft, this follows https://github.com/iodide-project/pyodide/issues/793#issue-738717762.

Closes https://github.com/iodide-project/pyodide/issues/793

### CODE REVIEW

**Comment by cglacet** [comment]
@rth  I will work on this tomorrow again, in the mean time let me know what you think about this.

**Comment by Alex-Monahan** [comment]
Quick question! Does Pyodide support using a SharedWorker instead of a Worker? That would be very helpful for my use case!

Thanks,
Alex

File: docs/using_pyodide_from_webworker.md [hunk 1]
```diff
@@ -10,83 +10,171 @@ Setup your project to serve `webworker.js`. You should also serve
 `pyodide.js`, and all its associated `.asm.js`, `.data`, `.json`, and `.wasm`
 files as well, though this is not strictly required if `pyodide.js` is pointing
 to a site serving current versions of these files.
+The simplest way to serve the required files is to use a CDN, 
+such as [pyodide-cdn2.iodide.io][cdn pyodide]. This is the solution
```
Comment (rth) [inline docs/using_pyodide_from_webworker.md:14]:
```suggestion
such as `https://cdn.jsdelivr.net/pyodide`. This is the solution 
```

There is probably need create a link there, as the index page will return 404 not found anyway.

File: docs/using_pyodide_from_webworker.md [hunk 2]
```diff
@@ -10,83 +10,171 @@ Setup your project to serve `webworker.js`. You should also serve
...
+`await pythonLoading;` as shown below:
+
+```js 
+// webworker.js 
+
+// Setup your project to serve `py-worker.js`. You should also serve
+// `pyodide.js`, and all its associated `.asm.js`, `.data`, `.json`, 
+// and `.wasm` files as well:
+self.languagePluginUrl = 'https://pyodide-cdn2.iodide.io/v0.15.0/full/';
+importScripts('https://pyodide-cdn2.iodide.io/v0.15.0/full/pyodide.js');
```
Comment (rth) [inline docs/using_pyodide_from_webworker.md:110]:
```suggestion
self.languagePluginUrl = 'https://cdn.jsdelivr.net/pyodide/v0.15.0/full/';
importScripts('https://cdn.jsdelivr.net/pyodide/v0.15.0/full/pyodide.js');
```

File: docs/using_pyodide_from_webworker.md [hunk 3]
```diff
@@ -10,83 +10,171 @@ Setup your project to serve `webworker.js`. You should also serve
...
-Call `postMessage` on your web worker, passing an object with the key `python`
-containing the script to execute as a string. You may pass other keys in the
-data object. By default the web worker assigns these to its global scope so that
-they may be imported from python. The results are returned as the `results` key,
-or if an error was encountered, it is returned in the `error` key.
+Communication from the worker to the main thread is done via the `Worker.postMessage()`
+method (and vice versa).
 
-For example:
+[cdn pyodide]: https://pyodide-cdn2.iodide.io/v0.15.0/full/
```
Comment (rth) [inline docs/using_pyodide_from_webworker.md:27]:
```suggestion

```

**Review by rth — APPROVED** [review]
Thank you @cglacet !
A few minor comments (mostly about CDN url) but otherwise LGTM.

**Pull request merged**